AddQtAndroidApk.cmake

```diff
@@ -44,8 +45,12 @@ if(NOT QT_ANDROID_NDK_ROOT)
     endif()
 endif()
 string(REPLACE "\\" "/" QT_ANDROID_NDK_ROOT ${QT_ANDROID_NDK_ROOT}) # androiddeployqt doesn't like backslashes in paths
+set(QT_ANDROID_NDK_ROOT ${QT_ANDROID_NDK_ROOT} CACHE STRING "" FORCE)
 message(STATUS "Found Android NDK: ${QT_ANDROID_NDK_ROOT}")
 
+set(QT_ANDROID_CMAKE_FOUND ON CACHE BOOL "QtAndroidCMake have been found" FORCE)
+set(QT_ANDROID_CMAKE_VERSION "19.0.1" CACHE STRING "QtAndroidCMake version" FORCE)
```

Comment:
What is that version? Where does it come from? Where is it used?

Reply:
I started to make git tag on some commits.
https://github.com/OlivierLDff/QtAndroidCMake/releases
I started v1 and then i tought it might be a good idea to match ndk version currently in usage. Seeing that now doesn't look like a good idea.
With that variable you could then print the version of the script included into your cmake logs. 
I don't think it's really usefull and the script shouldn't have data that can be found somewhere else. And i mean you should already be aware of the version you are using.